index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
```

Comment:
Pleas try to add comments for every section of you code. For better coding practice.